src/github.com/matrix-org/dendrite/common/config/config_test.go

```diff
@@ -42,9 +43,9 @@ media:
 kafka:
   addresses: ["localhost:9092"]
   topics:
-    input_room_event: input.room
     output_room_event: output.room
     output_client_data: output.client
+    user_updates: output.user
```

Comment:
Not strictly related to this PR, but the "input_room_event" is no longer a thing, but "user_updates" is a thing now.